docs/examples/customization/prompts/chat_prompts.ipynb

```diff
@@ -8,7 +8,6 @@
    ]
```

Comment:
<p>We don't need to import ChatMessage or MessageRole here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/11583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
fixed